src/main/java/com/cburch/logisim/std/memory/RandomHDLGeneratorFactory.java

```diff
@@ -54,133 +54,135 @@ public String getComponentStringIdentifier() {
+                          X"0000"&std_logic_vector(to_unsigned({{seed}}, 32));
+          s_reset      <= '1' WHEN s_reset_reg /= "010" ELSE '0';
+          s_reset_next <= "010" WHEN (s_reset_reg = "101" OR
+                                      s_reset_reg = "010") AND
+                                      clear = '0' ELSE
+                          "101" WHEN s_reset_reg = "001" ELSE
+                          "001";
+          s_start      <= '1' WHEN (ClockEnable = '1' AND enable = '1') OR
+                                   (s_reset_reg = "101" AND clear = '0') ELSE '0';
+          s_mult_shift_next <= (OTHERS => '0') WHEN s_reset = '1' ELSE
+                               X"5DEECE66D" WHEN s_start_reg = '1' ELSE
+                               '0'&s_mult_shift_reg(35 DOWNTO 1);
+          s_seed_shift_next <= (OTHERS => '0') WHEN s_reset = '1' ELSE
+                               s_current_seed WHEN s_start_reg = '1' ELSE
+                               s_seed_shift_reg(46 DOWNTO 0)&'0';
+          s_mult_busy       <= '0' WHEN s_mult_shift_reg = X"000000000" ELSE '1';
+          
+          s_mac_lo_in_1     <= (OTHERS => '0') WHEN s_start_reg = '1' OR
+                                                    s_reset = '1' ELSE
+                               '0'&s_mac_lo_reg(23 DOWNTO 0);
+          s_mac_lo_in_2     <= '0'&X"00000B"
+                                  WHEN s_start_reg = '1' ELSE
+                               '0'&s_seed_shift_reg(23 DOWNTO 0) 
+                                  WHEN s_mult_shift_reg(0) = '1' ELSE
+                               (OTHERS => '0');
+          s_mac_hi_in_2     <= (OTHERS => '0') WHEN s_start_reg = '1' ELSE
+                               s_mac_hi_reg;
+          s_mac_hi_1_next   <= s_seed_shift_reg(47 DOWNTO 24) 
+                                  WHEN s_mult_shift_reg(0) = '1' ELSE
+                               (OTHERS => '0');
+          s_busy_pipe_next  <= "00" WHEN s_reset = '1' ELSE
+                               s_busy_pipe_reg(0)&s_mult_busy;
+          
+          make_current_seed : PROCESS( GlobalClock , s_busy_pipe_reg , s_reset )
+          BEGIN
+             IF (GlobalClock'event AND (GlobalClock = '1')) THEN
+                IF (s_reset = '1') THEN s_current_seed <= s_InitSeed;
+                ELSIF (s_busy_pipe_reg = "10") THEN
+                   s_current_seed <= s_mac_hi_reg&s_mac_lo_reg(23 DOWNTO 0);
+                END IF;
+             END IF;
+          END PROCESS make_current_seed;
+          
+          make_shift_regs : PROCESS(GlobalClock,s_mult_shift_next,s_seed_shift_next,
+                                    s_mac_lo_in_1,s_mac_lo_in_2)
+          BEGIN
+             IF (GlobalClock'event AND (GlobalClock = '1')) THEN
+                s_mult_shift_reg <= s_mult_shift_next;
+                s_seed_shift_reg <= s_seed_shift_next;
+                s_mac_lo_reg     <= std_logic_vector(unsigned(s_mac_lo_in_1)+unsigned(s_mac_lo_in_2);
```

Comment:
add ```)``` before ```;```
It's a bit annoying, these were all errors that got already fixed for the 3.6.0